<ui version="4.0" >
 <class>detailWnd</class>
 <widget class="QDialog" name="detailWnd" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Vidéo </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLineEdit" name="leAdresse" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="pbRegarder" >
       <property name="text" >
        <string>&amp;Regarder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbEnregistrer" >
       <property name="text" >
        <string>&amp;Enregistrer : (hh:mm)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="teDuree" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>18</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbRetour" >
       <property name="text" >
        <string>R&amp;etour à la liste</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
